akka-docs/src/main/paradox/typed/routers.md

```diff
@@ -81,9 +81,19 @@ sent through the router, each actor is forwarded one message.
 Round robin gives fair routing where every available routee gets the same amount of messages as long as the set
 of routees stays relatively stable, but may be unfair if the set of routees changes a lot.
 
-This is the default for pool routers as the pool of routees is expected to remain the same. 
+This is the default for pool routers as the pool of routees is expected to remain the same.
+
+### Consistent Hashing
+ 
+Uses [consistent hashing](http://en.wikipedia.org/wiki/Consistent_hashing) to select a routee based
+on the sent message. This [article](http://www.tom-e-white.com/2007/11/consistent-hashing.html) 
+gives good insight into how consistent hashing is implemented.
+
+Currently you have to define hashMapping of the router to map incoming messages to their consistent
+hash key. This makes the decision transparent for the sender.
```

Comment:
Move this below Random as the other two are defaults for pool respective group, but this one always opt-in.

We need a sample snippet of implementing a good hash function here as well.